tests/Gaufrette/Functional/adapters/AzureMultiContainerBlobStorage.php.dist

```diff
@@ -0,0 +1,6 @@
+<?php
```

Comment:
We're deprecating this way of running functional tests. Could you rather implement `setUp` / `tearDow` methods to initialize the filesystem in your Test please? [Here's an example](https://github.com/KnpLabs/Gaufrette/blob/master/tests/Gaufrette/Functional/Adapter/AwsS3Test.php#L23).